### ISSUE TITLE
intel tensorflow 1.15.0up3 testing

### ISSUE THREAD

**Issue by yangw1234** [intel-analytics/analytics-zoo#4437]
1. bf16 model can be successfully trained
2. oneDNN

- [ ] bf16 verification
- [ ] make sure oneDNN ops can be turned off
- [ ] inception v1 example
- [ ] other examples, ut

**Comment by yangw1234** [comment]
@sgwhat

**Comment by sgwhat** [comment]
> @sgwhat

Roger that

**Comment by sgwhat** [comment]
In the tf2/resnet training data part, we'd better change the URL of instruction. The original tensorflow/model repository's slim bazel BUILD file is broken and not working, this [repository ](https://github.com/simnalamburt/models/tree/main/research/slim) updated BUILD and it works . @yangw1234

**Comment by yangw1234** [comment]
> In the tf2/resnet training data part, we'd better change the URL of instruction. The original tensorflow/model repository's slim bazel BUILD file is broken and not working, this [repository ](https://github.com/simnalamburt/models/tree/main/research/slim) updated BUILD and it works . @yangw1234

Let's use dummpy data first

**Comment by sgwhat** [comment]
> * [ ]  bf16 verification
> * [ ]  make sure oneDNN ops can be turned off
> * [x]  inception v1 example
> * [ ]  other examples, ut

**Comment by sgwhat** [comment]
| Task Name | Loss | Top1Accuracy | Running Time(s) |
| ------------ | ------------- | ------------- | ------------- |
| Basic_Text_Classification | 4.087298393249512 | 0.8654000163078308 | 151.16134071350098 |
| Transfer_Learning | 2.854679584503174 | 0.9340000152587891 | 355.7099766731262 |
